src/LibraryManager.Vsix/Contracts/Logger.cs

```diff
@@ -178,5 +179,147 @@ private static void LogOperationSummary(IEnumerable<ILibraryOperationResult> tot
                 LogEvent(messageText, LogLevel.Operation);
             }
         }
+
+        public static void LogErrors(IEnumerable<ILibraryOperationResult> results)
+        {
+            foreach (ILibraryOperationResult result in results)
+            {
+                if (result.Errors.Any())
```

Comment:
>if (result.Errors.Any()) [](start = 16, length = 24)

This isn't necessary. The foreach will bail if it's empty. Might need to check for null though?